packages/material-ui/src/Divider/Divider.test.js

```diff
@@ -65,4 +65,22 @@ describe('<Divider />', () => {
       });
     });
   });
+
+  describe('role', () => {
+    it('avoids adding implicit aria semantics', () => {
+      const wrapper = mount(<Divider />);
+      assert.strictEqual(wrapper.find('hr').props().role, undefined);
+    });
+
+    it('adds a proper role if none is specified', () => {
+      const wrapper = mount(<Divider component="div" />);
+      assert.strictEqual(wrapper.find('div').props().role, 'separator');
+    });
+
+    it('passed roles override', () => {
```

Comment:
I'm trying to think of a better way to word this.

Reply:
```suggestion
    test('passed roles override', () => {
```
?

Reply:
I guess, or `overrides the computed role with a provided one`